### PR TITLE
feat: commit from new branch

### PR DESCRIPTION
**Pull request by emilianoyaryura**

### CODE REVIEW

**Comment by julianbenegas** [comment]
@Emiliano0308 build is failing

File: components/sections/docs/docs-content.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        fontFamily: "special",
+      }}>
+      {children}
+    </h1>
+  );
+};
+
+export const SubHeading = ({ children }) => {
+  return (
+    <h1
```
Comment (julianbenegas) [inline components/sections/docs/docs-content.tsx:22]: We are hardcoding the `h1` here, which is incorrect

Comment (julianbenegas) [reply components/sections/docs/docs-content.tsx:22]: The editor should be able to put up to `h3`. The `h3` is the same but a bit smaller, and the margins tighter

**Comment by julianbenegas** [comment]
Weird white rectangle at the bottom on dark mode.

CleanShot 2021-03-10 at 14 29 51@2x

File: components/sections/docs/docs-mobile.nav.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+      <DocsMenu selected={selected} />
+      <div sx={{ position: "fixed", bottom: "100px", right: "16px" }}>
+        <DarkModeSwitch colorMode={colorMode} setColorMode={setColorMode} />
+      </div>
+    </div>
+  );
+};
+
+const DocsMobileNav = ({ setColorMode, colorMode, selected }) => {
+  const [menuOpen, setMenuOpen] = useState<boolean>(false);
```
Comment (julianbenegas) [inline components/sections/docs/docs-mobile.nav.tsx:60]: when the menu opens, document scroll must be disabled. See [this](https://github.com/basementstudio/lettermatic/blob/master/src/components/sections/header.tsx#L68-L103) for reference

**Comment by julianbenegas** [comment]
Fix console warnings

CleanShot 2021-03-10 at 14 35 10@2x

File: components/sections/docs/docs-menu.tsx [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+  }[];
+};
+
+const sections = [
+  {
+    title: "Video Application Developers",
+    links: [
+      {
+        title: "Overview",
+        href: "docs/video-application-developers/overview",
```
Comment (julianbenegas) [inline components/sections/docs/docs-menu.tsx:40]: Links should have `/` before, so they reference an absolute route. Now they are leading me into a 404

File: package.json [hunk 4]
```diff
@@ -47,7 +48,7 @@
     "@types/classnames": "^2.2.10",
     "@types/node": "^14.0.27",
     "@types/react": "^16.9.44",
-    "@types/theme-ui": "^0.3.6",
+    "@types/theme-ui": "^0.3.7",
```
Comment (julianbenegas) [inline package.json:51]: Maybe this is why we're having problems with theme-ui typings. Try downgrading to `0.3.1` (our current version of `theme-ui`, and see if it works well by removing our `index.d.ts`

File: pages/docs/[[...slug]].tsx [hunk 5]
```diff
@@ -12,23 +12,99 @@ import matter from "gray-matter";
...
+      }}>
+      <DocsNav
+        selected={realSlug}
+        setColorMode={setColorMode}
+        colorMode={colorMode}
+      />
+      <div
+        sx={{
+          display: ["none", "none", "flex"],
+          position: "fixed",
```
Comment (julianbenegas) [inline pages/docs/[[...slug]].tsx:74]: this should be `sticky`, not fixed

File: pages/docs/[[...slug]].tsx [hunk 6]
```diff
@@ -12,23 +12,99 @@ import matter from "gray-matter";
...
+import DocsMenu from "components/sections/docs/docs-menu";
+import NextStep from "components/sections/docs/next-step";
 
 type Params = { slug?: string[] };
 
 const Docs = ({
   mdx,
   meta,
+  path,
+  title,
```
Comment (julianbenegas) [inline pages/docs/[[...slug]].tsx:33]: Why not use `meta.title`?

**Review by julianbenegas — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (Emiliano0308) [reply package.json:51]: It didn't work

Comment (julianbenegas) [reply package.json:51]: oka

**Comment by julianbenegas** [comment]
Remember to re-request review when ready @emilianoyaryura

**Review by julianbenegas — APPROVED** [review]

**Pull request merged**